_sass/minimal-mistakes/_variables.scss

```diff
@@ -135,13 +135,16 @@ $small: 600px !default;
 $medium: 768px !default;
 $medium-wide: 900px !default;
 $large: 1024px !default;
+$large-wide: 1160px !default; 
 $x-large: 1280px !default;
+$max-width: 1280px !default;
```

Comment:
Having done a similar set of changes to my own fork of 'minimal-mistakes', perhaps this assignment should be written '$max-width: $x-large !default' so that overriding '$x-large' has the effect of also overriding the default value of 'max-width' as well.